packages/extension/src/ui/features/actions/connectDapp/ConnectDappScreen.tsx

```diff
@@ -306,15 +321,30 @@ export const ConnectDappScreen: FC<ConnectDappProps> = ({
           </>
         ) : (
           <>
-            <WarningText>No matched account found! Please create an account for: 
+            <WarningText>There are no accounts in the required group and network,
```

Comment:
[Nit] "There are no accounts on `testnet` for group `0`. Please generate a new account first.", WDYT?

Reply:
Sounds great, will update, thanks!